client/marshaller/kryo-marshaller/src/main/java/org/infinispan/marshaller/kryo/KryoTranscoder.java

```diff
@@ -0,0 +1,55 @@
+package org.infinispan.marshaller.kryo;
+
+import net.spy.memcached.CachedData;
+import net.spy.memcached.transcoders.Transcoder;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.0
+ */
+public class KryoTranscoder implements Transcoder<Object> {
```

Comment:
@galderz I don't remind removing the transcoders, but what is the harm of providing the user with a basic implementation if we will need it for testing anyway? Also, I believe @tristantarrant was a fan.